features/step_definitions/command_steps.rb

```diff
@@ -395,3 +395,13 @@
     end
   end
 end
+
+And (/^I create the "([^"]*)" bootstrap-repo for "([^"]*)" on the server$/) do |arch, target|
+  node = get_target(target)
+  os_version_raw, _code = node.run('grep "VERSION=" /etc/os-release')
+  os_version = os_version_raw.strip.split('=')[1].delete '"'
+  puts 'i should see the os version: ' + os_version
+  command = 'mgr-create-bootstrap-repo -c SLE-' + os_version + '-' + arch
+  puts 'running the command on the server:' + command
+  _out, $fail_code = $server.run(command, false)
```

Comment:
Why not? The previous step this command was running was https://github.com/SUSE/spacewalk-testsuite-base/blob/master/features/step_definitions/command_steps.rb#L214? Maybe it was generic handling for other tests that use this step?